### ISSUE TITLE
Control center color out of bounds

### ISSUE THREAD

**Issue by Cheesybiscuit** [Ominousness/MitsuhaXI#34]
When you look at the media player in the control center, the bottom half extends past the curves forming a square bottom.

**Comment by akswalia23** [comment]
Yes indeed it does and on 11.4 beta 3 mitsuha doesnt work in small CC only magnified/larger CC widget

**Comment by Ominousness** [comment]
Will be fixed in the next update.

**Comment by akswalia23** [comment]
Dude your awesome! I will be donating as soon as the update comes out
